react/src/pages/RoutingListPage.tsx

```diff
@@ -383,12 +410,39 @@ const RoutingListPage: React.FC<RoutingListPageProps> = () => {
                 <Suspense
                   fallback={<Spin indicator={<LoadingOutlined spin />} />}
                 >
-                  {endpoint?.model && (
-                    <VFolderLazyView uuid={endpoint?.model} clickable={false} />
-                  )}
+                  <Flex direction="column" align="start">
+                    <VFolderLazyView
+                      uuid={endpoint?.model as string}
+                      clickable={false}
+                    />
+                    {endpoint?.model_mount_destination && (
+                      <Flex direction="row" align="center" gap={'xxs'}>
+                        <ArrowRightOutlined type="secondary" />
+                        <Typography.Text type="secondary">
+                          {endpoint?.model_mount_destination}
+                        </Typography.Text>
+                      </Flex>
+                    )}
+                  </Flex>
                 </Suspense>
               ),
             },
+            {
+              label: t('modelService.AdditionalMounts'),
+              children:
+                (endpoint?.extra_mounts?.length as number) > 0 ? (
+                  _.map(endpoint?.extra_mounts, (vfolder) => (
+                    <VFolderLazyView
+                      uuid={vfolder?.row_id as string}
+                      clickable={false}
+                    />
+                  ))
+                ) : (
```

Comment:
`endpoint.extra_mounts` has a name of vfolder, so we can display the name directly instead of using `VFolderLazyView`. `VFolderLazyView` sends a request to get the name by ID.